plugin.video.vtm.go/service.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+""" Service entry point """
+
+from __future__ import absolute_import, division, unicode_literals
+
+from resources.lib import service
+
+service.run()
```

Comment:
since the addon already includes a service maybe you can move the player object there and avoid sleeping after requesting playback.